.buildbot.sh

```diff
@@ -35,6 +35,14 @@ cd tests
 	cargo test
 cd .. 
 
+# Test the handling of `extern` variables of type `const * const` when joining
```

Comment:
Can't we just do `make emit-test-extern-const-constptr` here or does that not do these steps?

Reply:
It would another long line like the one above with all the variables.

Reply:
Sorry, I don't understand.

Reply:
We emit the code in the Makefile and add other steps in .buildbot.sh, because the call to `./split-llvm-extract` did not work in the Makefile even with all the env variables set.

Reply:
If the `Makefile` doesn't work, then there's a problem we should fix there, surely?

Reply:
The other tests are never run via the Makefile, that's why I had to do this workaorund here.

Reply:
> Can't we just do `make emit-test-extern-const-constptr` here or does that not do these steps?

Fixed in 7291c28.